plutus-core/plutus-core/test/Evaluation/Spec.hs

```diff
@@ -1,11 +1,247 @@
+{-# LANGUAGE AllowAmbiguousTypes   #-}
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE LambdaCase            #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE PolyKinds             #-}
+{-# LANGUAGE TypeApplications      #-}
+{-# LANGUAGE TypeFamilies          #-}
+{-# LANGUAGE TypeOperators         #-}
+
 module Evaluation.Spec where
 
+import Control.Monad.Except
+import Data.ByteString qualified as BS
+import Data.Functor ((<&>))
+import Data.Int (Int64)
+import Data.List.Extra qualified as List
+import Data.Text (Text)
+import Data.Type.Equality
+import Data.Typeable (splitTyConApp)
 import Evaluation.Machines (test_machines)
-
+import Hedgehog hiding (Opaque, Var, eval)
+import Hedgehog.Gen qualified as Gen
+import Hedgehog.Range qualified as Range
+import PlutusCore hiding (Term)
+import PlutusCore qualified as PLC
+import PlutusCore.Builtin
+import PlutusCore.Data (Data (..))
+import PlutusCore.Generators
+import Prettyprinter
 import Test.Tasty
+import Test.Tasty.Hedgehog
+import Type.Reflection
+import Unsafe.Coerce
+
+type Term = PLC.Term TyName Name DefaultUni DefaultFun ()
+
+test_builtinsDon'tThrow :: TestTree
+test_builtinsDon'tThrow =
+  testGroup
```

Comment:
Another nitpick: we more or less consistently use four spaces for indentation. I wouldn't bother reindenting this file, but in future please do follow the convention (I had to learn to do it too).

Reply:
Updated.